Library/ViewModels/DashboardViewModel.swift

```diff
@@ -87,25 +102,65 @@ public protocol DashboardViewModelType {
 public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewModelOutputs,
   DashboardViewModelType {
 
-    public init() {
+  //We use producers ot this properties during view appearance and projects retreaval process 
```

Comment:
Would you please proof this comment? There are some typos and I'm not quite following.

We don't typically add extra producers/properties like this that aren't associated with direct inputs. Are there other ways we could write this logic? Thoughts @theginbin @justinswart?